### PR TITLE
SITES-515 Fix double title for nodes in editorial

### PR DESCRIPTION
**Pull request by maxious**
As per review feedback on SITES-515.
Also fix name of Editorial section.

### CODE REVIEW

File: app/views/editorial/nodes/show.html.haml [hunk 1]
```diff
@@ -1,6 +1,3 @@
-= content_for :title do
-  = node.name
-
```
Comment (tessereth) [inline app/views/editorial/nodes/show.html.haml]: Maybe you could add a comment explaining why this file doesn't have a `content_for :title`? Then if someone comes by later and notices it's not there, they won't add it by accident. And it we change the way previews are handled, we know we might need to add it back.

**Comment by tessereth** [comment]
Other than one minor thing, LGTM.

**Comment by bjgordon** [comment]
LGTM

**Comment by maxious** [comment]
rebase'd

**Pull request merged**